roles/openshift_metrics/defaults/main.yaml

```diff
@@ -59,3 +59,6 @@ openshift_metrics_cassandra_pvc_access: "{{ openshift_hosted_metrics_storage_acc
 openshift_metrics_hawkular_user_write_access: False
 
 openshift_metrics_heapster_allowed_users: system:master-proxy
+
+openshift_metrics_cassandra_enable_prometheus_endpoint: True
```

Comment:
@mwringe ack, would that be a scenario at installation or post installation? I'm just trying to figure out if this should be a single variable or two. If they are mutually exclusive then it makes sense to keep them as two.